packages/snap/src/rpc/vc/createVC.ts

```diff
@@ -0,0 +1,48 @@
+import { CreateVCRequestParams } from '@blockchain-lab-um/ssi-snap-types';
+import { copyable, divider, heading, panel, text } from '@metamask/snaps-ui';
+import { W3CVerifiableCredential } from '@veramo/core';
+import { ApiParams } from 'src/interfaces';
+
+import { snapConfirm } from '../../utils/snapUtils';
+import { veramoCreateVC, veramoSaveVC } from '../../utils/veramoUtils';
+
+export async function createVC(
+  params: ApiParams,
+  createVCParams: CreateVCRequestParams
+): Promise<W3CVerifiableCredential> {
+  const {
+    minimalUnsignedCredential,
+    proofFormat = 'jwt',
```

Comment:
I think we don't need the `= 'jwt'`, because we do the same in the `veramoCreateVC` function